manager/src/main/java/org/openremote/manager/map/MapService.java

```diff
@@ -127,6 +140,20 @@ protected static Metadata getMetadata(Connection connection) {
         return metadata;
     }
 
+    @SuppressWarnings("unchecked")
+    protected static ConcurrentMap<String, MapRealmConfig> getMapConfigRealms(ObjectNode mapRealms){
+        return (ConcurrentMap<String, MapRealmConfig>) ValueUtil.JSON.convertValue(mapRealms, ConcurrentMap.class);
+    }
+
+    protected static ObjectNode loadMapSettingsJson() {
+        return configurationService.getMapConfig();
+    }
+
+    @SuppressWarnings("unchecked")
+    protected ConcurrentMap<String, ObjectNode> getMapSettings(){
+        return (ConcurrentMap<String, ObjectNode>) ValueUtil.JSON.convertValue(this.mapConfig, ConcurrentMap.class);
```

Comment:
It's used when calling `HTTP GET /map`, and I don't really understand what happens there, since it dynamically builds the Mapbox GL settings from mapsettings.json